azure-pipelines/templates/steps/vsts-release.yml

```diff
@@ -0,0 +1,80 @@
+# File: azure-pipelines\templates\steps\nightly-snapshot-releases\vsts-release.yml
+# Description: Template to assemble and publish Snapshot/Latest versions of ${{ parameters.projectName }}
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# Variable: 'folderName' refers to the project folder
+# Variable: 'gradleTasks' list all the gradle tasks to be used.
+# Variable: 'accessTokenName'  secret used to access the private azure devops maven feed.
+# Variable: 'runUnitTest' boolean: refers to whether the pipeline will run unit tests or not.
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'versionType' type of version (Latest, Snapshot)
+
+parameters:
+- name: versionType
+  default: ''
+  values:
+  - Snapshot
+  - Latest
+- name: projectName
+  default: none
+- name: gradleTasks
+  default: none
+- name: accessTokenName
+  default: none
+- name: runUnitTest
+  default: false
+
+jobs:
+- job: Phase_1
+  displayName: Initiating Release Process
+  timeoutInMinutes: 90
+  cancelTimeoutInMinutes: 1
+  condition: and(ne(variables['parameters.versionType'], ''), ne(variables['parameters.gradleTasks'], ''))
```

Comment:
Looks like now the entire pipeline is going to run based on condition that versionType is specified? that's probably not what we want...I think we should be able to repurpose this pipeline for regular VSTS releases that aren't of the snapshot or latest type...hence I was asking to make the `version update` task conditional (not the entire pipeline)